### ISSUE TITLE
Loading drivers from plugins directory

### ISSUE THREAD

**Issue by kurumpa** [metabase/metabase#9059]
Hi. 
I'm trying to implement a driver and load it from plugins directory. But no matter whether I put it under `./plugins` or setting `MB_PLUGINS_DIR`, I'm gettings results similar to #8676. After digging a little bit I've found out that: 
1) `(plugins/load-plugins!)` -> `(dynamically-add-jars!)` -> `(dynapath/add-classpath-url sysloader (.toURL (.toURI jar-file)))` works normal, 
but 
2) `(classpath/system-classpath)` at https://github.com/metabase/metabase/blob/master/src/metabase/util.clj#L484 yields only `metabase.jar`. So I'm currently bypassing this by changing `(classpath/system-classpath)` to `(classpath/loader-classpath (ClassLoader/getSystemClassLoader))` (as the file URI is there, after step 1) )

I am not very familiar with Closure/Java stack, I'd really appreciate if you could tell me what I am possibly doing wrong. Thank you.

My setup is `java version "1.8.0_152"` at `macOS 10.14.1`, debugging in `IntelliJ IDEA 2018.2.5`

**Comment by camsaul** [comment]
Hi @kurumpa, I'm currently working on overhauling the way we load drivers from the plugins directory. 

In the next few weeks or so I expect to have the changes merged into master which should make this a lot easier. Sit tight!

**Comment by salsakran** [comment]
@kurumpa  @camsaul did the refactor resolve this issue?

**Comment by flamber** [comment]
Closing, since modular drivers has been a thing since 0.32
